packages/constants/src/lib/constants/constants.ts

```diff
@@ -492,9 +520,12 @@ export const LIT_CHAINS: LITChain<LITEVMChain> = {
     contractAddress: null,
     chainId: 175177,
     name: 'Chronicle - Lit Protocol Testnet',
-    symbol: 'testLPX',
+    symbol: 'tstLPX',
```

Comment:
I think this one is still off -- 
![image](https://github.com/user-attachments/assets/0fc7584a-8004-4885-8e00-dcbd17225855)

```suggestion
    symbol: 'tstLIT',
```